### ISSUE TITLE
A11y: Tab order lands on element "Pane" with no focus indicator

### ISSUE THREAD

**Issue by chingucoding** [files-community/Files#2919]
**Describe the bug**
When opening a folder and tabbing through the UI, I noticed that focus can land on an element that is being announced "Pane" without a more descriptive name that also has no focus visual:

**To Reproduce**
1. Open folder in Files (in my case desktop)
2. Tab through entire app
3. Start narrator
4. Notice elements visited and announced
4. Notice pane without name or focus indicator

**Expected behavior**
Either give it a proper name and add focus indicator or skip element in tab order
**Screenshots**
![Gif showing faulty tab order where focus rect moves through app and then suddenly disappears](https://user-images.githubusercontent.com/16122379/104227019-051c0400-5449-11eb-8dd0-f339542368ee.gif)

**Desktop (please complete the following information):**
 - OS Version: Win10 10.0.19042
 - App version: [1.0.0.0]

**Additional context**
Add any other context about the problem here. Does this problem occur again after restarting the app?

**Log file**
Please upload the log file here so that we can understand your issue better. You can access it from Settings->About->Open log location. You can also find the log file manually by following [this](https://github.com/files-community/Files/wiki/Opening-the-log-file) guide.
[debug_fulltrust.txt](https://github.com/files-community/Files/files/5797991/debug_fulltrust.txt)
[debug.txt](https://github.com/files-community/Files/files/5797992/debug.txt)

**Comment by yaichenbaum** [comment]
@chingucoding This issue should be resolved with #3312.
